lib/fileops/file_ops_test.go

```diff
@@ -61,35 +59,35 @@ func TestGlob(t *testing.T) {
 
 	tsmfile := files[0:2]
 	sort.Strings(tsmfile)
-	tmpFiles, err := Glob("/tmp/test/*.tsm")
+	tsmPattern := path.Join(tmpDir, "*.tsm")
+	tmpFiles, err := Glob(tsmPattern)
 	if err != nil {
-		t.Fatalf("Glob(/tmp/test/*.tsm) failed: %q", err.Error())
+		t.Fatalf("Glob(%s) failed: %q", tsmPattern, err.Error())
 	}
 	if len(tmpFiles) != len(tsmfile) {
-		t.Fatalf("Glob(/tmp/test/*.tsm) failed")
+		t.Fatalf("Glob(%s) failed", tsmPattern)
 	}
 	for i := 0; i < len(tmpFiles); i++ {
-		if tsmfile[i] != tmpFiles[i] {
-			t.Fatalf("Glob(/tmp/test/*.tsm) failed: expected(%s) but(%s)", tsmfile[i], tmpFiles[i])
+		if path.Join(tmpDir, tsmfile[i]) != tmpFiles[i] {
```

Comment:
Glob() may not have the files in lexical order, so it is possible that tsmfile[i] and tmpFiles[i]
may mismatch on some operating systems. Same for walfiles[] below.